pkg/operator/configobserver/network/observe_network.go

```diff
@@ -14,46 +15,116 @@ import (
 func GetClusterCIDRs(lister configlistersv1.NetworkLister, recorder events.Recorder) ([]string, error) {
 	network, err := lister.Get("cluster")
 	if errors.IsNotFound(err) {
-		recorder.Warningf("ObserveRestrictedCIDRFailed", "Required networks.%s/cluster not found", configv1.GroupName)
+		recorder.Warningf("GetClusterCIDRs", "Required networks.%s/cluster not found", configv1.GroupName)
```

Comment:
You're inconsistent about "`GetClusterCIDRs`" vs "`GetClusterCIDRsFailed`" (and likewise in the other functions).

Reply:
I was trying to match style (that's a Reason, not an event category or source, but there's no clear distinction.